PathToRegex/PathToRegex.swift

```diff
@@ -20,7 +20,12 @@
 //===----------------------------------------------------------------------===//
 
 import Foundation
-import Regex
+
+#if COCOAPODS
+    import CrossroadRegex
```

Comment:
Hey @pgherveou , I have finally managed to go through [Regex](https://github.com/crossroadlabs/Regex) framework and apply the changes. Still, there are major breaking changes (Xcode 8... you know) in release 0.7, so I will need some time to adjust PathToRegex as well.

Thanks for your patience!